### PR TITLE
#7 #8 change project structure, added section Services and Clients

### PR DESCRIPTION
**Pull request by IlyaKirsanov**

### CODE REVIEW

File: src/components/OurClients/ArrowComp/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+
+import arrowRightBig from '../../../assets/svg/button-arrow-right-48-px.svg';
+
+function ArrowComp(props) {
```
Comment (RadzivilVlad) [inline src/components/OurClients/ArrowComp/index.js:5]: export default class ArrowComp extends React.PureComponent

File: src/components/OurClients/Testimonial/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+const  Testimonial = (props) => {
```
Comment (RadzivilVlad) [inline src/components/OurClients/Testimonial/index.js:3]: export default class Testimonial extends React.PureComponent

File: src/components/OurClients/index.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+					infinite: true,
+					slidesToShow: 1,
+					slidesToScroll: 1,
+					initialSlide: 0
+				}
+			}]		
+		};
+
+
+		let testimonials = (
```
Comment (RadzivilVlad) [inline src/components/OurClients/index.js:39]: const

File: src/components/OurClients/index.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+							name={testimonial.name}
+							position={testimonial.position}
+							comment={testimonial.comment}
+						/>
+					})}
+				</Slider>
+			</div>
+		);
+
+		let logos = (
```
Comment (RadzivilVlad) [inline src/components/OurClients/index.js:55]: const

File: src/components/OurServices/ArrowComp/index.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+import arrowRightBig from '../../../assets/svg/button-arrow-right-48-px.svg';
+
+function ArrowComp(props) {
+	const {onClick } = props;
```
Comment (BonBuasson) [inline src/components/OurServices/ArrowComp/index.js:5]: Can you add space?
`const {onClick } = props;` => `const { onClick } = props;`

File: src/components/OurServices/ArrowComp/index.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+import arrowRightBig from '../../../assets/svg/button-arrow-right-48-px.svg';
+
+function ArrowComp(props) {
```
Comment (RadzivilVlad) [inline src/components/OurServices/ArrowComp/index.js:4]: code duplications

File: src/components/OurServices/index.js [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+					slidesToShow: 2,
+					slidesToScroll: 1,
+					initialSlide: 0
+				}
+			},
+			]
+		};
+
+
+		let services = (
```
Comment (RadzivilVlad) [inline src/components/OurServices/index.js:57]: const services = ...

File: src/components/OurServices/index.js [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+							image={service.image}
+							title={service.title}
+							description={service.description}
+						/>
+					})}
+				</Slider>
+			</div>
+		);
+
+		let teams = (
```
Comment (RadzivilVlad) [inline src/components/OurServices/index.js:72]: const

File: src/components/OurServices/ServiceCard/index.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react';
+import arrowRightSmall from '../../../assets/svg/button-arrow-right-40-px.svg';
+
+const ServiceCard = (props) =>{
```
Comment (RadzivilVlad) [inline src/components/OurServices/ServiceCard/index.js:4]: export default class ServiceCard extends React.PureComponent

File: src/components/OurServices/ServiceCard/index.js [hunk 10]
```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react';
```
Comment (RadzivilVlad) [inline src/components/OurServices/ServiceCard/index.js:1]: import React from 'react';

File: src/components/OurServices/TeamCard/index.js [hunk 11]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const TeamCard = (props) => {
```
Comment (RadzivilVlad) [inline src/components/OurServices/TeamCard/index.js:3]: export default class TeamCard extends React.PureComponent

File: src/components/OurServices/index.js [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+import ServiceCard from './ServiceCard'
+import data from '../../data/data'
+import arrowRightBig from '../../assets/svg/button-arrow-right-48-px.svg';
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import TeamCard from './TeamCard';
+import ArrowComp from './ArrowComp';
+
+class OurServices extends Component {
```
Comment (RadzivilVlad) [inline src/components/OurServices/index.js:11]: React.Component

File: src/components/OurServices/index.js [hunk 13]
```diff
@@ -0,0 +1,113 @@
+import React, { Component, PureComponent } from 'react';
```
Comment (RadzivilVlad) [inline src/components/OurServices/index.js:1]: import React from 'react';

File: src/components/OurClients/index.js [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+			<div className="clients-logos">
+				<ul className="logos-list">
+					{this.state.logos.map(logo => {
+						return (
+							<li className="logos-list__item">
```
Comment (BonBuasson) [inline src/components/OurClients/index.js:60]: `key` is missed in child component
`<li className="logos-list__item">..</li>`
    =>
`<li key={someKey} className="logos-list__item">..</li>`

File: src/components/OurClients/index.js [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+								<div className="logo-container">
+									<img key={logo.id} src={logo.logo} alt="logo" />
```
Comment (BonBuasson) [inline src/components/OurClients/index.js:62]: remove `key`
`logo`
   =>
`logo`

File: src/components/OurServices/index.js (hunk 12 above)
Comment (BonBuasson) [inline src/components/OurServices/index.js:11]: `export default class OurServices extends React.PureComponent`

File: src/components/OurClients/ArrowComp/index.js (hunk 1 above)
Comment (IlyaKirsanov) [reply src/components/OurClients/ArrowComp/index.js:5]: done

File: src/components/OurClients/Testimonial/index.js (hunk 2 above)
Comment (IlyaKirsanov) [reply src/components/OurClients/Testimonial/index.js:3]: done

File: src/components/OurClients/index.js (hunk 3 above)
Comment (IlyaKirsanov) [reply src/components/OurClients/index.js:39]: done

File: src/components/OurClients/index.js (hunk 4 above)
Comment (IlyaKirsanov) [reply src/components/OurClients/index.js:55]: done

File: src/components/OurServices/ArrowComp/index.js (hunk 6 above)
Comment (IlyaKirsanov) [reply src/components/OurServices/ArrowComp/index.js:4]: done

File: src/components/OurServices/index.js (hunk 7 above)
Comment (IlyaKirsanov) [reply src/components/OurServices/index.js:57]: done

File: src/components/OurServices/ServiceCard/index.js (hunk 9 above)
Comment (IlyaKirsanov) [reply src/components/OurServices/ServiceCard/index.js:4]: done

File: src/components/OurServices/index.js (hunk 8 above)
Comment (IlyaKirsanov) [reply src/components/OurServices/index.js:72]: done

File: src/components/OurServices/ServiceCard/index.js (hunk 10 above)
Comment (IlyaKirsanov) [reply src/components/OurServices/ServiceCard/index.js:1]: done

File: src/components/OurServices/TeamCard/index.js (hunk 11 above)
Comment (IlyaKirsanov) [reply src/components/OurServices/TeamCard/index.js:3]: done

File: src/components/OurServices/index.js (hunk 12 above)
Comment (IlyaKirsanov) [reply src/components/OurServices/index.js:11]: done

File: src/components/OurServices/index.js (hunk 13 above)
Comment (IlyaKirsanov) [reply src/components/OurServices/index.js:1]: done

File: src/components/OurServices/ArrowComp/index.js (hunk 5 above)
Comment (IlyaKirsanov) [reply src/components/OurServices/ArrowComp/index.js:5]: done

File: src/components/OurClients/index.js (hunk 14 above)
Comment (IlyaKirsanov) [reply src/components/OurClients/index.js:60]: done

File: src/components/OurClients/index.js (hunk 15 above)
Comment (IlyaKirsanov) [reply src/components/OurClients/index.js:62]: done

File: src/components/OurServices/index.js (hunk 12 above)
Comment (IlyaKirsanov) [reply src/components/OurServices/index.js:11]: done

**Pull request merged**